### ISSUE TITLE
[PE-D][Tester E] App - [done mod] Tasks not marked as done after marking module as done the second time

### ISSUE THREAD

**Issue by nus-se-script** [AY2223S1-CS2103T-W10-4/tp#193]
### Steps to reproduce

1. Launch the app
2. Type in the command `done mod cs2109s`
3. Type in the command `add task Re-mod -c cs2109s`
4. Type in the command `done mod cs2109s`

### Expected

One of the following:
* Mark the task as done after step 4
* Error message: Unable to add task to a module that is already done

### Actual

Tasks not being marked as done.

### Screenshot
`add task Re-mod -c cs2109s`
![image.png](https://raw.githubusercontent.com/yuxuanleong/ped/main/files/4780e483-a70d-4686-96fc-54cb1310b611.png)
`done mod cs2109s`
![image.png](https://raw.githubusercontent.com/yuxuanleong/ped/main/files/3f511dc9-9a8d-47dc-bb05-c9be22ae3efa.png)

-------------
Labels: `severity.Low` `type.FunctionalityBug`
original: yuxuanleong/ped#6

**Comment by jontmy** [comment]
Was able to reproduce the bug:

https://user-images.githubusercontent.com/50949210/198817918-a23f5294-553e-4255-b3b1-722e3c0e9fcd.mov

Marking the task as done via `done task` works as a workaround, but `done mod` needs to be fixed.
